.github/workflows/ci.yml

```diff
@@ -48,7 +48,49 @@ jobs:
       - run: pip install flake8 flake8-print
       - run: flake8 --enable=T modin
 
+  prepare-cache:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-latest",  "windows-latest"]
+        python-version: ["3.6", "3.7", "3.8"]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Cache conda
+        uses: actions/cache@v1
+        with:
+          path: ~/conda_pkgs_dir
+          key: ${{ matrix.os }}-python-${{ matrix.python-version }}-conda-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
+      - name: Cache pip if Ubuntu
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
why are you not using the check from documentation like `if: startsWith(runner.os, 'Linux')`? here and for Windows

Reply:
Here I can only answer: 'because my way works too' :)

Reply:
No problem to change.